lib/internal/errors.js

```diff
@@ -69,6 +91,7 @@ module.exports = exports = {
   Error: makeNodeError(Error),
   TypeError: makeNodeError(TypeError),
   RangeError: makeNodeError(RangeError),
+  AssertionError: AssertionError,
```

Comment:
just `AssertionError,` should suffice (it does not need the additional `: AssertionError` bit)